src/commands/actor/index.ts

```diff
@@ -1,8 +1,7 @@
 import { ApifyCommand } from '../../lib/apify_command.js';
 
 export class ActorIndexCommand extends ApifyCommand<typeof ActorIndexCommand> {
-	static override description =
-		'Commands are designed to be used in Actor runs. All commands are in PoC state, do not use in production environments.\n';
+	static override description = 'Manages runtime data operations inside of a running Actor.\n';
```

Comment:
some descriptions have the trailing `\n` but some don't (e.g. `actors pull`). it looks weird to have it there, i can imagine it will get trimmed by oclif.